### DISCUSSION TITLE
【qiita-cli 機能追加の要望】qiita pull 先のrootのディレクトリ名を指定できるようにしてほしい

### DISCUSSION THREAD

**Discussion by TakuKobayashi** [increments/qiita-discussions#514] [要望・アイデア・不具合]
[qiita-cli](https://github.com/increments/qiita-cli) にて `qiita pull` を実行した時に現在は `public/` 以下に記事が自動的に作成されます。 `qiita pull` のオプションが現在(version 0.7.0)では `-f` `--force` しかないため、pull 先のrootディレクトリ名を `public/` 以外に指定することができません。
そこで `-r` `--root-path` あるいは `-p` `--project` などのようなオプション機能を追加して `qiita pull` をおこなった時の root ディレクトリを指定できるようにしてほしいと思います。

**Comment by ohakutsu** [comment]
@TakuKobayashi 
ご意見ありがとうございます！

現在、`--root`オプションを使うことで記事ファイルがダウンロードされるディレクトリを指定できます。

```
$ npx qiita pull --root ./my_articles/
```

以下のような構造で作成されるのですが、この方法で今回いただいたご要望は解決できますでしょうか？

```
└── my_articles/
   └── public/
      └── 記事ファイル1.md
      ├── 記事ファイル2.md
      └── 記事ファイル3.md
```

**Comment by TakuKobayashi** [comment]
@ohakutsu
ありがとうございます!!
はい、これでしたら私の要望は解決できます
本要望はcloseといたします
